src/ILVerify/src/ILVerify.csproj

```diff
@@ -1,19 +1,38 @@
-<Project Sdk="Microsoft.NET.Sdk">
-
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFrameworks>netcoreapp2.0;net46</TargetFrameworks>
     <PlatformTarget>AnyCPU</PlatformTarget>
     <CLSCompliant>false</CLSCompliant>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
+    <TargetFramework>netstandard1.5</TargetFramework>
```

Comment:
I expect that it won't be possible to debug this project in Visual Studio with these changes. We may want copy the secret sauce from `src\ILCompiler\src\ILCompiler.csproj` to make the project debuggable in Visual Studio. The relevant parts are:

```
    <TargetFramework>netcoreapp2.0</TargetFramework>
    <CopyNugetImplementations>false</CopyNugetImplementations>
    <!-- Force .dll extension even if output type is exe. -->
    <TargetExt>.dll</TargetExt>
```

```
  <ItemGroup>
    <Content Include="ilc.runtimeconfig.json">
      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
    </Content>
  </ItemGroup>
```

```
  <PropertyGroup>
    <RunCommand Condition="'$(OS)' == 'Windows_NT'">$(TargetDir)$(AssemblyName).exe</RunCommand>
    <RunCommand Condition="'$(OS)' != 'Windows_NT'">$(TargetDir)$(AssemblyName)</RunCommand>
    <RunArguments>$(StartArguments)</RunArguments>
  </PropertyGroup>
```